storage/s3_test.go

```diff
@@ -699,3 +678,82 @@ func TestS3listObjectsV2(t *testing.T) {
 	}
 	assert.Equal(t, len(mapReturnObjNameToModtime), 0)
 }
+
+func TestSessionCreateAndCachingWithDifferentRegions(t *testing.T) {
+	const bucketRegionPath = "CreateBucketConfiguration.LocationConstraint"
+
+	testcases := []struct {
+		region         string
+		expectedRegion string // to check if `create session` request with specific region was executed
+
+		alreadyCreated bool // sessions should not be created again if they already have been created before
+	}{
+		{},
```

Comment:
does an empty test have a meaning?

Reply:
It is used to create session with default region, for when it is not provided. should we add comments or make it more explicit?